### PR TITLE
dummy deps + recipes for scorep/30rc1

### PR DESCRIPTION
**Pull request by jgphpc**
dummy deps for scorep (SLES12)

### CODE REVIEW

File: easybuild/easyconfigs/c/Cube/Cube-4.3.4-debug.eb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+versionsuffix = '-debug'
+
+homepage = 'http://www.scalasca.org/software/cube-4.x/download.html'
+description = """Cube, which is used as performance report explorer for Scalasca and
+ Score-P, is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system
+ resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree
+ can be collapsed or expanded to achieve the desired level of granularity."""
+
+# toolchain = {'version': '2015.11-XC', 'name': 'CrayGNU'}
```
Comment (gppezzi) [inline easybuild/easyconfigs/c/Cube/Cube-4.3.4-debug.eb]: Do you need to keep all these comments?

File: easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.eb [hunk 2]
```diff
@@ -1,19 +1,16 @@
-##
 # This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
-#
-# Copyright:: Copyright 2013-2016 Juelich Supercomputing Centre, Germany
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
```
Comment (gppezzi) [inline easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.eb]: Aren't you overwriting a recent file with an older one?

File: easybuild/easyconfigs/o/OTF2/OTF2-2.0.eb [hunk 3]
```diff
@@ -1,40 +1,42 @@
-##
 # This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
-# Copyright:: Copyright 2013-2016 Juelich Supercomputing Centre, Germany
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
```
Comment (gppezzi) [inline easybuild/easyconfigs/o/OTF2/OTF2-2.0.eb]: same here, looks like you're using an old verison.

**Comment by gppezzi** [comment]
Can you please provide instructions on where and how to build this PR? 

Should we do module purge before building the dummy .ebs?

PR name suggests is only dummy deps for Score-P but you also adds other recipes (PGI, CCE, ...)

File: easybuild/easyconfigs/c/Cube/Cube-4.3.4-debug.eb (hunk 1 above)
Comment (jgphpc) [inline easybuild/easyconfigs/c/Cube/Cube-4.3.4-debug.eb]: yes.

File: easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.eb (hunk 2 above)
Comment (jgphpc) [inline easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.eb]: maybe but is this an issue ?

Comment (gppezzi) [inline easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.eb]: if you reviewed the diffs, not really.

File: easybuild/easyconfigs/c/Cube/Cube-4.3.4-debug.eb (hunk 1 above)
Comment (gppezzi) [inline easybuild/easyconfigs/c/Cube/Cube-4.3.4-debug.eb]: I can understand that you want to keep the comments below, but the one on line 26 doesn't make much sense to me.

We can merge like this if you want, but this looks more like WIP.

**Comment by jgphpc** [comment]
Waiting for deps [rpms](https://webrt.cscs.ch/Ticket/Display.html?id=24322) to be installed on brisi LN.

**Comment by jgphpc** [comment]
i will create new separate PR because this one is not clear enough.

**Pull request closed without merging**